modules/utxo-lib/src/bitgo/parseInput.ts

```diff
@@ -11,16 +11,57 @@ export function isPlaceholderSignature(v: number | Buffer): boolean {
   return v === 0;
 }
 
-export interface ParsedSignatureScript {
+/**
+ * @return Checks if P2TR script path's control block matches BitGo's need
+ */
+export function isValidControlBock(controlBlock: Buffer): boolean {
+  return Buffer.isBuffer(controlBlock) && 33 <= controlBlock.length && controlBlock.length % 32 === 1;
+}
+
+/**
+ * @return Calculates script path level for P2TR control block.
+ */
+export function calculateScriptPathLevel(controlBlock: Buffer): number | undefined {
```

Comment:
```suggestion
export function getScriptPathLevel(controlBlock: Buffer): number | undefined {
```

Reply:
we could also throw here and simplify the return value

Reply:
Done